source/server/management/performance-tweaks.rst

```diff
@@ -87,16 +105,30 @@ lower this interval, then change it in the servers ``global.conf`` file:
 
 .. code-block:: none
 
-  world {
+    world {
         # The auto-save tick interval used when saving global player data.
         # Set to 0 to disable. (Default: 900) Note: 20 ticks is equivalent to 1 second.
         auto-player-save-interval=900
 
         # The auto-save tick interval used to save all loaded chunks in a world.
         # Set to 0 to disable. (Default: 900) Note: 20 ticks is equivalent to 1 second.
         auto-save-interval=900
-        }
+    }
 
 Reducing this interval increases the load on your server's CPU and storage, but reduces the data loss that might occur
 if the server locks up or the power fails. Conversely, increasing the auto-save interval reduces the load on the
 hardware, but at the expense of increasing the amount of in-game progress that could be lost in case of server failure.
+
+Realtime
+========
+
+Enabling this will not actually improve performance, but instead will improve your experience a lot when tick rate is low. A limited set of entities, tile entities and world time will use real time instead of ticks to update.
+
+.. code-block:: none
+
+    modules {
+        # Use real (wall) time instead of ticks as much as possible
+        realtime=false
+    }
+
+One example of this is baby animals. Normally, they take 20 minutes to grow into an adult. However, if the server is lagging, each animal will receive fewer ticks thus increasing the time they take to grow up. This setting updates some of their logic to use the actual elapsed wall-clock time, rather than number of ticks. It will also apply to block breaking, so no more "breaking blocks multiple times".
```

Comment:
Add newlines to avoid overly long lines in the editor. 120chars max.

Reply:
✅ done